src/main.ts

```diff
@@ -30,6 +32,26 @@ async function run(): Promise<void> {
       }
     }
 
+    if (downloadSettings.addToPathEnvironmentVariable) {
+      const out = downloadSettings.outFilePath;
+      // Make executables executable
+      for (const file of readdirSync(out)) {
+        let full = join(out, file);
+        const toSliceTo = /-[0-9]/.exec(file);
+        if (toSliceTo) {
+          const old = full;
+          full = join(out, file.slice(0, toSliceTo.index));
+          renameSync(old, full);
+          core.debug(`Renamed ${old} to ${full}`);
+        }
+        const newMode = statSync(full).mode | constants.S_IXUSR | constants.S_IXGRP | constants.S_IXOTH;
+        chmodSync(full, newMode);
+        core.info(`Added ${full} executable`);
+      }
+      core.addPath(out);
+      core.info(`Added ${out} to PATH`);
+    }
+
```

Comment:
It would be nice if this functionality is covered by some tests.

Reply:
I think I have done this now 👍  https://github.com/robinraju/release-downloader/pull/732/files#diff-9e111cd57e3607f69c6194fe85699c3408c950526a2cd1b8695b3cc6208a0977